cmd/margo/serve_e2e_test.go

```
@@ -0,0 +1,157 @@
+		}
+		time.Sleep(10 * time.Millisecond)
+	}
+	t.Fatalf("timed out waiting for serving URL: %s", output.String())
+	return ""
+}
+
+func waitForServedContent(t *testing.T, url, required string) {
+	t.Helper()
+	deadline := time.Now().Add(5 * time.Second)
+	for time.Now().Before(deadline) {
+		response, err := http.Get(url)
```

golangci-lint (2.12.2)

[error] net/http.Get must not be called. use net/http.NewRequestWithContext and (*net/http.Client).Do(*http.Request)

(noctx)